anki-plugin/src/trmodels.py

```diff
@@ -326,6 +326,10 @@ class TiddlyRememberClozeTemplate(TemplateData):
         back = """
             {{cloze:Text}}
 
+            <br>
+
+            {{Extra}}
+
```

Comment:
To match Anki's built-in cloze note type:

```suggestion
            {{cloze:Text}}<br>
            {{Extra}}
```